gnat2goto/COMPILING.md

```diff
@@ -27,7 +27,7 @@ select your platform and then 2016 (in that order). Download
   * gnatcoll-gpl-2016
 
 
-The following assumes you installed GNAT GPL 2016 into /opt/gnat using the doinstall script.
+Install the GNAT GPL Ada 2016 using the `doinstall` script inside the zip. The following instructions assume you picked `/opt/gnat` as the install destination. 
```

Comment:
I don’t think the doinstall script is strictly necessary, but I suppose if you didn’t want to use it you could just ignore this so OK.

Reply:
The recommended way to install it is using the doinstall script - that's following AdaCore's instructions. In CI, we install it slightly differently via a make command, but only because that's better suited to automated install - and it's not a documented way to install it. Ahem.